### PR TITLE
more renamings of acronyms to camelCase

### PR DESCRIPTION
**Pull request by erik-krogh**
One last round of renaming acronyms to follow our style guide. I'm not planning any more renamings of upper-case acronyms after this.   

The [change-notes from last time](https://github.com/github/codeql/blob/main/java/ql/lib/change-notes/2022-08-22-xml-rename.md) still work. 

The tests fail, but there is an internal PR where they pass.

### CODE REVIEW

**Review by smowton — DISMISSED** [review]
Go :+1:

**Review by esbena — APPROVED** [review]
JS 👍🏻

**Review by michaelnebel — APPROVED** [review]
C# 👍

**Review by jketema — APPROVED** [review]
C++ :+1:

**Review by aschackmull — APPROVED** [review]

**Review by aschackmull — APPROVED** [review]
Java 👍

File: python/ql/test/experimental/dataflow/TestUtil/NormalDataflowTest.qll [hunk 1]
```diff
@@ -31,3 +31,6 @@ query predicate missingAnnotationOnSINK(Location location, string error, string
     )
   )
 }
+
+/** DEPRECATED: Alias for missingAnnotationOnSink */
+deprecated predicate missingAnnotationOnSINK = missingAnnotationOnSink/3;
```
Comment (RasmusWL) [inline python/ql/test/experimental/dataflow/TestUtil/NormalDataflowTest.qll:36]: This is just a test-file, so we can just do the rename without deprecation :blush: 

```suggestion
```

**Review by RasmusWL — APPROVED** [review]
Python :+1: overall

File: python/ql/test/experimental/dataflow/TestUtil/NormalDataflowTest.qll (hunk 1 above)
Comment (erik-krogh) [reply python/ql/test/experimental/dataflow/TestUtil/NormalDataflowTest.qll:36]: I'll delete the deprecation with my deprecation-deletion-script in 14+ months anyway.  
And I don't want to run the tests again, and bump the submodule pointer again, so I'll let this one stay.

**Pull request merged**